.github/workflows/conformance-clustermesh.yaml

```diff
@@ -0,0 +1,504 @@
+name: ClusterMesh
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  issue_comment:
+    types:
+      - created
+  # Run every 6 hours
+  schedule:
+    - cron:  '0 3/6 * * *'
+  ### FOR TESTING PURPOSES
+  # This workflow runs in the context of `master`, and ignores changes to
+  # workflow files in PRs. For testing changes to this workflow from a PR:
+  # - Make sure the PR uses a branch from the base repository (requires write
+  #   privileges). It will not work with a branch from a fork (missing secrets).
+  # - Uncomment the `pull_request` event below, commit separately with a `DO
+  #   NOT MERGE` message, and push to the PR. As long as the commit is present,
+  #   any push to the PR will trigger this workflow.
+  # - Don't forget to remove the `DO NOT MERGE` commit once satisfied. The run
+  #   will disappear from the PR checks: please provide a direct link to the
+  #   successful workflow run (can be found from Actions tab) in a comment.
+  #
+  pull_request: {}
```

Comment:
any reason to plan running this job with pull_request_target ? AFAIK, there is no secret used in this workflow :thinking:

Reply:
Actually the idea was keep the same approach as the existing one, and run the tests on `issue_command` (and schedule) only. This is consistent with the approach adopted also in the conformance-datapath workflow, as both spawn a quite large number of jobs. I'm obviously open to change the approach if you feel running on pull_request would be better.

Reply:
thanks for clarification :100: